wire-ios-data-model/Tests/Source/Model/Conversation/ZMConversationTests+SecurityLevel.swift

```diff
@@ -519,10 +519,10 @@ class ZMConversationTests_SecurityLevel: ZMConversationTestsBase {
             selfUser.name = "MYSELF"
             conversation.addParticipantAndUpdateConversationState(user: selfUser, role: nil)
 
-            let mainTeam = Team.fetchOrCreate(with: UUID.create(),
-                                              create: true,
-                                              in: self.syncMOC,
-                                              created: nil)!
+            let mainTeam = Team.fetchOrCreate(
+                with: UUID.create(),
```

Comment:
Is it possible here to use the default init instead of the customised func?
```suggestion
                with: UUID,
```

Reply:
yes, done c20f80c1b73c1667c7d15bf5a50a7831398ed8da